cmd/clusterctl/README.md

```diff
@@ -66,9 +66,9 @@ Once you have created a cluster, you can interact with the cluster and machine
 resources using kubectl:
 
 ```
-$ kubectl --kubeconfig kubeconfig get clusters
-$ kubectl --kubeconfig kubeconfig get machines
-$ kubectl --kubeconfig kubeconfig get machines -o yaml
+$ kubectl get clusters
+$ kubectl get machines
+$ kubectl get machines -o yaml
```

Comment:
I thought by default we will use ~/.kube/config ? or it will fall back to kubeconfig on $PWD?